header.php

```diff
@@ -15,7 +15,7 @@
 <head>
 	<meta charset="<?php bloginfo( 'charset' ); ?>">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<link rel="profile" href="http://gmpg.org/xfn/11">
+	<link rel="profile" href="https://gmpg.org/xfn/11">
```

Comment:
Can this be set to `//gmpg.org/xfn/11` (protocol relative URL)? This way you cover both `http` and `https` (locally you usually don't have any kind of certificate set up).

Reply:
Sorry @dingo-d I don't agree that protocol relative URL is the best solution. The HTTPS link will always work regardless if your URL is HTTPS or not. https://www.paulirish.com/2010/the-protocol-relative-url/

Reply:
@grappler is right.

Reply:
@grappler @philiparthurmoore Great! I've removed the commit, that leaves us with `https`. Good to merge :)